.github/workflows/scripts/issues/titleCleanup.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check
+const regexes = {
+  // covers 'dataGrid' in all variants, as well as with suffixes (-pro/-premium/etc)
+  dataGrid: /\s?\[\s?(data\s?grid[-\sA-Za-z]*)]/gi,
+  // covers 'charts' in all variants, as well as with prefixes (PieChart, LineChart, etc)
+  charts: /\s?\[\s?([-\sA-Za-z]*charts?)]/gi,
+  // covers 'pickers' in all variants, as well as with prefixes (DatePicker, time-picker, etc)
+  pickers: /\s?\[\s?([-\sA-Za-z]*pickers?)]/gi,
+  // covers 'treeView' in all variants
+  treeView: /\s?\[\s?([-\sA-Za-z]*tree\s?(view|item))]/gi,
+};
+
+const replacements = {
+  dataGrid: 'data grid',
+  charts: 'charts',
+  pickers: 'pickers',
+  treeView: 'tree view',
+};
+
+const regex =
+  /\[\s*(?:(?<dataGrid>data\s?grid.*)|(?<charts>.*charts?)|(?<pickers>.*pickers?)|(?<treeView>tree\s?(view|item)))\s*]/gi;
```

Comment:
What a monstrosity, 10/10 would buy again

Reply:
It doesn't seem to be used though :(

Reply:
Oh... it slipped in... that was the first iterration! 😂